packages/core/card.ts

```diff
@@ -412,13 +463,10 @@ export class Card {
     }
   }
 
-  private async serialize(
-    rules: OcclusionRulesOrDefaults,
-    included: Map<string, OcclusionRulesOrDefaults[]>
-  ): Promise<ResourceObject> {
+  private async serialize(rules: OcclusionRules, included: Map<string, OcclusionRules[]>): Promise<ResourceObject> {
     let data = Object.create(null);
 
-    if (this instanceof AddressableCard) {
```

Comment:
This doesn't really work, because sometimes we're an instance of FieldCard.

(Also we were sometimes an instance of Card, but I addressed that by adding a `patch` method to `FieldCard`so that patching a FieldCard gives another FieldCard and not a Card.)